index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html> 
+
+<html lang="en">
+      <head>
```

Comment:
your html doesn't look like on screenshot you provided. I guess you forgot to push your latest changes